src/modules/fancyzones/FancyZonesLib/FancyZonesWindowProcessing.cpp

```diff
@@ -27,8 +27,9 @@ bool FancyZonesWindowProcessing::IsProcessable(HWND window) noexcept
 
     // popup could be the window we don't want to snap: start menu, notification popup, tray window, etc.
     // also, popup could be the windows we want to snap disregarding the "allowSnapPopupWindows" setting, e.g. Telegram
-    bool isPopup = FancyZonesWindowUtils::IsPopupWindow(window) && !FancyZonesWindowUtils::HasThickFrameAndMinimizeMaximizeButtons(window);
-    if (isPopup && !FancyZonesSettings::settings().allowSnapPopupWindows)
+    bool isPopup = FancyZonesWindowUtils::IsPopupWindow(window);
+    bool hasThickFrame = FancyZonesWindowUtils::HasThickFrame(window);
+    if (isPopup && !(hasThickFrame && FancyZonesSettings::settings().allowSnapPopupWindows))
```

Comment:
```suggestion
    if (isPopup && (!hasThickFrame || !FancyZonesSettings::settings().allowSnapPopupWindows))
```
The current way it's written is a bit confusing in my opinion. I've had to use Morgan's Theorem to make some sense of what this line meant. I suggest changing it like this to make it a bit more easy to reason about. WDYT?

Reply:
Yeah, looks good to me. The result is the same but seems to be more readable. Thanks!